### DISCUSSION TITLE
Impressive project! Financial support?

### DISCUSSION THREAD

**Discussion by ViktorThink** [turboderp/exllama#119] [General]
Hey @turboderp 

I'm really impressed by the repo and would be happy to support it if you choose to accept donations!

I'm having some struggles with a setup for batch generation, and would be happy to donate $300 usd for an hour of your time. Hopefully you could get something out of the call as well. Totally get if you don't have time/don't want, in which case I'm sure I can find another way.

Keep up the great work!

**Comment by turboderp** [comment]
I'm currently really busy at work and won't have a whole lot of time this project, for the next week or so. After that, well, it's a generous enough offer but I can't really think of this as a completed project that I can start helping people put to use like that. And honestly, I'm kinda looking more towards ExLlamaV2 right now, consolidating what works best in V1 and making room for some new features.

I'd be happy to weigh in other discussions in the meantime, though.

**Reply by ViktorThink** [reply]
I totally understand and want you to know that the work you've done with exllama is amazing, and look forward to seeing ExLlamaV2!

If you however do get the time feel free to ping here or at discord (same username there, ViktorThink).
